apps/emqx_bridge/test/emqx_bridge_api_SUITE.erl

```diff
@@ -612,6 +612,7 @@ t_enable_disable_bridges(Config) ->
         <<"{\"code\":\"BAD_REQUEST\",\"message\":\"Forbidden operation, bridge not enabled\"}">>,
         Res
     ),
+    {ok, 400, Res} = request(post, operation_path(cluster, start, BridgeID), <<"">>),
```

Comment:
Just a small suggestion: I tried to see in this test (`t_enable_disable_bridges`) if there was an assertion to check that `?BRIDGE_NOT_FOUND/404` is returned for this operation, but I didn't spot it.  Maybe it would be interesting to add a case for that?

Reply:
Yeah, I might have missed to add one here and there. I'd rather have a cleanup or rather refactoring ticket and make this more composable. 

Current status is that things are very repetitive without clear patterns even. Be it tests or application logic. It's far from ideal. And kinda beyond the scope of this ticket.